lib/controller.js

```diff
@@ -113,6 +118,11 @@ JWTController.prototype.keycloakIntrospect = function(payload, data, token, call
       payload.scope = _.uniq(payload.scope.concat(data.defaultScopes));
     }
 
+    const tokenScope = _.get(payload, 'scope', null);
+    if (typeof tokenScope === 'string') {
+      payload.scope = payload.scope.split(' ');
```

Comment:
whats the format of scope? I thought it was comma delimited?

Reply:
Example: `scope: user company-read company-write`
This is with newest keycloak update

Reply:
got it!